fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java

```diff
@@ -172,7 +172,8 @@ private StorageVolume getStorageVolumeOfTable(String svName, long dbId) throws D
             } else {
                 sv = getStorageVolumeByName(BUILTIN_STORAGE_VOLUME);
                 if (sv == null) {
-                    throw new DdlException("Builtin storage volume not exists, please check the params in config");
+                    throw new DdlException("Cannot find a suitable storage volume. " +
+                            "The possible cause is that 'enable_load_volume_from_conf' is set to false during the upgrade.");
```

Comment:
So what should users do?

Reply:
You can get the value of  `Config.enable_load_volume_from_conf` here and provide a clearer prompt.

Reply:
@sduzh PTAL